en/docs/learn/vscode-plugin-samples/database-polling.md

```diff
@@ -0,0 +1,66 @@
+
+## Setup database to run the sample
+
+1. Create a database and a user with the following details and grant that user access to the database.
+
+    ```bash
+    CREATE DATABASE hospital_db;
+    USE hospital_db;
+    CREATE USER 'user'@'localhost' IDENTIFIED BY 'password';
+    GRANT ALL PRIVILEGES ON hospital_db.* TO 'user'@'localhost';
+    ```
+
+2. Create a table and insert some records.
+
+    ```bash
+    CREATE TABLE `doctors` (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `name` varchar(50) DEFAULT NULL,
+    `hospital` varchar(50) DEFAULT NULL,
+    `speciality` varchar(50) DEFAULT NULL,
+    `availability` varchar(50) DEFAULT NULL,
+    `charge` int(10) DEFAULT NULL,
+    `sync` int(11) DEFAULT '0',
+    PRIMARY KEY (`id`)
+    );
+    ```
+
+    ```bash
+    INSERT INTO doctors (name, hospital, speciality, availability, charge) values
+    ('thomas collins', 'grand oak community hospital', 'surgery', '9.00 a.m - 11.00 a.m', 7000),
+    ('henry parker', 'grand oak community hospital', ' ent', '9.00 a.m - 11.00 a.m', 4500),
+    ('abner jones', 'grand oak community hospital', 'gynaecology', '8.00 a.m - 10.00 a.m', 11000),
+    ('abner jones', 'grand oak community hospital', 'ent', '8.00 a.m - 10.00 a.m', 6750),
+    ('anne clement', 'clemency medical center', 'surgery', '8.00 a.m - 10.00 a.m', 12000),
+    ('thomas kirk', 'clemency medical center', 'gynaecology', '9.00 a.m - 11.00 a.m', 8000),
+    ('cailen cooper', 'clemency medical center', 'paediatric', '9.00 a.m - 11.00 a.m', 5500),
+    ('seth mears', 'pine valley community hospital', 'surgery', '3.00 p.m - 5.00 p.m', 8000),
+    ('emeline fulton', 'pine valley community hospital', 'cardiology', '8.00 a.m - 10.00 a.m', 4000),
+    ('jared morris', 'willow gardens general hospital', 'cardiology', '9.00 a.m - 11.00 a.m', 10000),
+    ('henry foster', 'willow gardens general hospital', 'paediatric', '8.00 a.m - 10.00 a.m', 10000),
+    ('chris brown', 'pine valley community hospital', 'surgery', '9.00 a.m - 11.00 a.m', 5500),
+    ('victor ivan', 'pine valley community hospital', 'ent', '9.00 a.m - 11.00 a.m', 5500),
+    ('jimmy carter', 'willow gardens general hospital', 'ent', '9.00 a.m - 11.00 a.m', 2500),
+    ('peter simon', 'clemency medical center', 'cardiology', '9.00 a.m - 11.00 a.m', 6000);
+    ```
+
+## Deploying the sample
+
+1. Open the sample by clicking on the **Database Polling** card.
+2. Give a folder location to save the sample.
+3. Open the `DoctorsDataService` data service artifact and update the RDBMS datasource in it by replacing the `URL`, `Username` and `Password` values with your database credentials.
+4. Download the `mysql-connector-j-8.0.32.jar` and add it to the `<PROJECT_WORKSPACE>/deployment/libs` folder.
```

Comment:
Let's add the download link for the JAR file here.

Reply:
Fixed in 8844ccd775aa7132437584a8e0baee0ad4e5982a